beacon_node/network/src/sync/sampling.rs

```diff
@@ -71,7 +71,12 @@ impl<T: BeaconChainTypes> Sampling<T> {
                 self.log.clone(),
             )),
             Entry::Occupied(_) => {
-                warn!(self.log, "Ignoring duplicate sampling request"; "id" => ?id);
+                // Sampling is triggered from multiple sources, duplicate sampling requests are
+                // likely (gossip block + gossip data column)
+                // TODO(das): Should track failed sampling request for some time? Otherwise there's
+                // a risk of a loop with multiple triggers creating the request, then failing,
```

Comment:
I think the main one we want to avoid retrying is `TooManyFailures` right? so it doesn't reset the required success to the min number? ([`IncementalDAS`](https://ethresear.ch/t/lossydas-lossy-incremental-and-diagonal-sampling-for-data-availability/18963#incrementaldas-dynamically-increase-the-sample-size-10))